modules/rosa-basic-concepts.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// understanding-rosa/understanding-rosa.adoc
+
+
+[id="rosa-basic-concepts"]
+= Basic Concepts
```

Comment:
Sentence case here, too, however could this intro bit (the following sentence) just be put directly into the `getting_started_rosa/basic-concepts-rosa.adoc` assembly, and then the following subsections be broken out as modules? That way it also wouldn't introduce the extra mid-level heading, which doesn't seem to add much (i.e., it's just the same title as the assembly title).

Reply:
Done.